Tests/MarkdownKitTests/InlineTests.swift

```diff
@@ -187,4 +187,15 @@ final class InlineTests: XCTestCase {
             NSRange(location: 6, length: 1), NSRange(location: 25, length: 1)
         ])
     }
+
+    func testMail() {
+        let markdown = "<foo@bar.com>"
+        let document = Parser.parse(markdown)!
```

Comment:
Usually opposed to a hard `!` in a test since if one thing goes out, the whole test suite comes crashing down.